### PR TITLE
util: add tests for util package

### LINKED ISSUE #193: util module is not being tested
Currently the `util` module is not covered by any tests. We should start adding at least some basic ones, which would help avoiding breaking things (e.g. https://github.com/coreos/go-systemd/pull/192) during refactoring.

### PR DESCRIPTION
**Pull request by hodgesds**
add tests for the util module

Fixes #193

Not sure if all the assertions are correct on the return values, but it does add a bit of coverage.

### CODE REVIEW

**Comment by lucab** [comment]
ok to test

**Comment by lucab** [comment]
Thanks for the testcases, @hodgesds! Just a couple of comments:
 * can you please add a copyright header on top of it?
 * I think this test should be gated behind a `cgo` buildtag, they are not much useful without it
 * Our test runners are fine with it, but I think those tests should exit gracefully if systemd is missing in the build/test environment. Can we short-circuit and skip when `!IsRunningSystemd`?

**Comment by hodgesds** [comment]
>can you please add a copyright header at the top of the file?
Will do!

>I think these tests should be gated behind a cgo buildtag, they are not much useful without it
Makes sense.

>Our test runners are fine with it, but I think those tests should exit gracefully if systemd is missing in the build/test environment. Can we short-circuit and skip when !IsRunningSystemd?
I wasn't sure on the desired behavior but I like that approach.

Will try to get an updated PR out later tonight, thanks!

File: util/util_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	s, err := CurrentUnitName()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	if s == "" {
+		t.Error("CurrentUnitName returned a empty string")
+	}
+}
+
+func TestIsRunningSystemd(t *testing.T) {
```
Comment (jonboulle) [inline util/util_test.go]: Bit confused about the point of this one...

**Comment by lucab** [comment]
Sorry for the delay, LGTM.

**Pull request merged**